src/std/fs.ab

```diff
@@ -76,3 +78,33 @@ pub fun change_owner(user: Text, path: Text): Bool {
 
     return false
 }
+
+/// Escapes all characters in the passed-in glob except "*", "?" and "/",
+/// to prevent injection attacks.
+fun escape_non_glob_chars(path: Text): Text {
+    return replace_regex(path, "\([^*?/]\)", "\\\\\1")
+}
+
+/// Finds all files or directories matching multiple file globs. When
+/// we have union types, this functionality can be merged into the main
+/// `glob` function.
+pub fun glob_multiple(paths: [Text]): [Text]? {
+    let combined = ""
+    if len(paths) == 1 {
+        combined = escape_non_glob_chars(paths[0])
+    } else {
+        let items = [Text]
+        loop item in paths {
+            item = escape_non_glob_chars(item)
+            items += [item]
+        }
+        combined = join(items, " ")
+    }
```

Comment:
We could remove that `if..else` block and leave the content from the `else` block... right?

```suggestion
        let items = [Text]
        loop item in paths {
            item = escape_non_glob_chars(item)
            items += [item]
        }
        combined = join(items, " ")
```

Reply:
We could, but then we'd be concatenating a single string, which seems redundant and slightly inefficient.

Reply:
But I can see the benefit in keeping it simple, too.

Reply:
I see, @hdwalters... How about this then? We remove the nested block making it easier to read.


```suggestion
    if len(paths) == 1:
        return escape_non_glob_chars(paths[0])

    let items = [Text]
    loop item in paths {
        item = escape_non_glob_chars(item)
        items += [item]
    }
    combined = join(items, " ")
```

Reply:
You can't do an early return there, the function has to work with the `combined` variable after creating it. In any case, this is just 3 levels of indentation so I don't see a problem.

Reply:
Oh I didn't see this part of the function when I checked on it
```
    let files = $eval "for file in {combined}; do [ -e \\\"\\\$file\\\" ] && echo \\\"\\\$file\\\"; done"$?
    return split(files, "\n")
```